packages/eslint-plugin/docs/rules/quotes.md

```diff
@@ -19,4 +24,6 @@ It adds support for TypeScript features which allow quoted names, but not backti
 
 See [`eslint/quotes` options](https://eslint.org/docs/rules/quotes#options).
 
-<sup>Taken with ❤️ [from ESLint core](https://github.com/eslint/eslint/blob/master/docs/rules/quotes.md)</sup>
+:::note
+Taken with ❤ [from ESLint core](https://github.com/eslint/eslint/blob/master/docs/rules/quotes.md)
+:::
```

Comment:
is there any way to not show the `:::note` on github? It makes it look pretty garbage
![image](https://user-images.githubusercontent.com/7462525/110027411-c850e880-7ce6-11eb-95a7-532d18edee54.png)

Reply:
no there is not

Reply:
Can you add double new line, double end of line space, or `<br />` on the special lines without breaking it?
Maybe we can at least put that on newlines to make it look a little nicer?

Reply:
we can insert new line between :::note and words, i'm going to do that